tests/rules/assertions/arrayStyleSimpleType.js

```diff
@@ -128,6 +128,16 @@ export default {
           onlyFilesWithFlowAnnotation: true
         }
       }
+    },
+
+    // While this isn't valid flow, we shouldn't disallow it.
```

Comment:
What is the use case for this if it isn't valid Flow?

Reply:
Mostly just a sanity check to check that the code won't break. I'm not attached to it at all.